invenio_rdm_records/records/api.py

```diff
@@ -77,6 +77,12 @@ class CommonFieldsMixin:
             pid_field=Vocabulary.pid.with_type_ctx('languages'),
             cache_key='languages',
         ),
+        resource_type=PIDRelation(
+            'metadata.resource_type',
+            attrs=['id', 'title'],  # TODO?: 'props'
```

Comment:
Vocabulary records have an `id`, `title` and `props` fields. The `props` field contains all the additional correspondences between formats for resource types in this case (e.g. the `datacite_general`, `datacite_type`), but since we were able to access those fields when we needed them (in `resources/serializers/datacite/schema.py`) directly, there is no need to add `props` just yet I think.